.eslintrc.js

```diff
@@ -1,4 +1,4 @@
-const dictionary = require('./.eslint-dictionary.json');
+const dictionary = require("./.eslint-dictionary.json");
```

Comment:
The majority of the code base prefers single quotes, can we stick to that?

Reply:
Yes. I will add prettier to the ensemble here so it can enforce quotations. The default formatter configured with VSCode does a poor job of that.